src/main/kotlin/racingcar/domain/racing/CarsHistory.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar.domain.racing
+
+import racingcar.domain.car.Car
+import racingcar.domain.car.Cars
+
+class CarsHistory(private val carsHistory: MutableList<Cars>) : List<Cars> by carsHistory {
+
+    val latest: Cars? get() = carsHistory.getOrNull(carsHistory.lastIndex)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    val latest: Cars? get() = carsHistory.lastOrNull()
```

Reply:
https://github.com/next-step/kotlin-racingcar/pull/238/commits/64124d9c155b513b8bff051a2dcb6b50b682f72a 좋아요!